src/viam/sdk/spatialmath/geometry.cpp

```diff
@@ -38,8 +38,17 @@ viam::common::v1::Pose GeometryConfig::pose_proto() const {
     return pose;
 }
 
-GeometryConfig GeometryConfig::from_proto(viam::common::v1::Geometry proto) {
+GeometryConfig GeometryConfig::from_proto(const viam::common::v1::Geometry& proto) {
     GeometryConfig cfg;
+    auto& pose = proto.center();
```

Comment:
Oh, it looks like we'll need to explicitly declare this as `const` or the clang-tidy will get mad.